setup.cfg

```diff
@@ -22,24 +22,22 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-    sphinx>=3,<5
+    myst-nb>=0.13,<0.18
```

Comment:
@AakashGfude 👍

Reply:
do we require any specific version of `sphinx`?

Reply:
@mmcky  that is being handled by myst-nb, and matches with this extensions requirement.

Reply:
Ah - makes sense 👍